node/src/main/resources/swagger-ui/swagger.json

```diff
@@ -4250,674 +4980,2182 @@
+            "additionalProperties": {
               "type": "array",
               "items": {
+                "required": [
+                  "name",
+                  "type"
+                ],
                 "type": "object",
                 "properties": {
-                  "type": {
-                    "type": "string",
-                    "description": "integer",
-                    "enum": [
-                      "integer",
-                      "boolean",
-                      "binary",
-                      "string"
-                    ]
-                  },
-                  "key": {
+                  "name": {
                     "type": "string"
                   },
-                  "value": {
-                    "type": "object"
+                  "type": {
+                    "type": "string"
                   }
-                },
-                "required": [
-                  "key",
-                  "type",
-                  "value"
-                ]
+                }
               }
-            },
-            "transfers": {
-              "type": "array",
+            }
+          }
+        }
+      },
+      "BinaryEntry": {
+        "required": [
+          "key",
+          "type",
+          "value"
+        ],
+        "type": "object",
+        "properties": {
+          "key": {
+            "type": "string"
+          },
+          "type": {
+            "type": "string",
+            "enum": ["binary"]
+          },
+          "value": {
+            "type": "string",
+            "format": "byte"
+          }
+        }
+      },
+      "BooleanEntry": {
+        "required": [
+          "key",
+          "type",
+          "value"
+        ],
+        "type": "object",
+        "properties": {
+          "key": {
+            "type": "string"
+          },
+          "type": {
+            "type": "string",
+            "enum": ["boolean"]
+          },
+          "value": {
+            "type": "boolean"
+          }
+        }
+      },
+      "DataEntry": {
+        "required": [
+          "key",
+          "type",
+          "value"
+        ],
+        "type": "object",
+        "properties": {
+          "type": {
+            "type": "string",
+            "enum": [
+              "integer",
+              "boolean",
+              "binary",
+              "string"
+            ]
+          },
+          "key": {
+            "type": "string"
+          },
+          "value": {
+            "type": "object",
+            "properties": {},
+            "description": "integer/boolean/binary/string value"
+          }
+        }
+      },
+      "DeleteEntry": {
+        "required": [
+          "key",
+          "value"
+        ],
+        "type": "object",
+        "properties": {
+          "key": {
+            "type": "string"
+          },
+          "value": {
+            "type": "object",
+            "properties": {},
+            "description": "null"
```

Comment:
null without "

Reply:
"null" here is a comment (I've added a more detailed explanation now)